docs/containers-sigstore-signing-params.yaml.5.md

```diff
@@ -0,0 +1,117 @@
+% CONTAINERS-SIGSTORE-SIGNING-PARAMS.YAML 5 sigstore signing parameters Man Page
+% Miloslav Trmač
+% January 2023
+
+# NAME
+containers-sigstore-signing-params.yaml - syntax for the sigstore signing parameter file
+
+# DESCRIPTINO
+
+Sigstore signing parameter files are used to store options that may be required to create sigstore signatures.
+There is no standard location for these files; they are user-managed, and used as inputs to a container image signing operation,
```

Comment:
On the fence, your call.  "standard location" or "default location"?

Reply:
(I’m not very sure that sentence even needs to exist… I just wanted to give _some_ context for users who stumble on the page directly, without starting at a `--sign-by-sigstore` option documentation.)

Sure, “default location” works.